lib/.npmignore

```diff
@@ -0,0 +1,2 @@
+src/
+!/dist/types/src
```

Comment:
```suggestion
!/dist/types/src

```

Do we need ! here?

Reply:
Yes as /src would exclude all directory with the src name, so we need to exclude src inside types